### ISSUE TITLE
Static Markdown Site Mode

### ISSUE THREAD

**Issue by tolicodes** [11ty/eleventy#1352]
I have a repo full of interlinked markdown files. All I want to do is serve them up. However I experienced a lot of issues with linking and missing some files.

I had to:

Create a `.eleventy.js` file in the root

```
module.exports = function (eleventyConfig) {
  const markdownIt = require('markdown-it')({
    replaceLink: function (link) {
      return link.replace('.md', '/');
    }
  }).use(require('markdown-it-replace-link'));

  eleventyConfig.setLibrary("md", markdownIt);
};
```

Install the markdown packages
```
yarn add markdown-it markdown-it-replace-link
```

Use a different build command:
```
eleventy --passthroughall --input content
```

If there any easier way to do this

**Comment by zachleat** [comment]
Yeah I don’t think this will work very well with `permalink`.

Have you seen the link shortcode example at https://github.com/11ty/eleventy/issues/544#issuecomment-496752551

**Comment by zachleat** [comment]
> This repository is now using [lodash style issue management](https://twitter.com/samselikoff/status/991395669016436736) for enhancements. This means enhancement issues will now be closed instead of leaving them open.

> View [the enhancement backlog here](https://github.com/11ty/eleventy/issues?utf8=%E2%9C%93&q=label%3Aneeds-votes+sort%3Areactions-%2B1-desc+). Don’t forget to upvote the top comment with 👍!

**Comment by binyamin** [comment]
This could be fixed with #84

**Comment by zachleat** [comment]
https://www.11ty.dev/docs/plugins/inputpath-to-url/ is the shipped feature that solves this!
